frontend/app/app.js

```diff
@@ -17,5 +17,3 @@ const App = Application.extend({
 });
 
 loadInitializers(App, config.modulePrefix);
-
-export default App;
```

Comment:
@trbmcginnis the error you're hitting with the `test-helpers.js` file is because you're importing Application from this file, but currently this file isn't exporting anything (there's no `default` that can be brought in!).  If you add this line back into the file it clears the error.

Reply:
Awesome, thank you!!!